spec/controllers/api/encryptables_controller_spec.rb

```diff
@@ -10,7 +10,9 @@
   let(:api_user) { bob.api_users.create }
   let(:private_key) { bob.decrypt_private_key('password') }
   let(:nested_models) { ['folder'] }
-  let(:attributes) { %w[name cleartext_password cleartext_username] }
+  # rubocop:disable Metrics/LineLength
```

Comment:
do not add an exception here ... rubocop is gränning because the line is really toooooo long :wink: 

use do end block and put arguments on mutlipe lines